content/azure_portal.md

```diff
@@ -14,265 +14,12 @@ aliases = ["/azure_portal.html"]
 
 [\[edit on GitHub\]](https://github.com/chef/chef-web-docs/blob/master/content/azure_portal.md)
 
-{{% cloud_azure_portal %}}
-
-## Virtual Machines running Chef Infra Client
-
 {{< warning >}}
 
-When the Chef VM extension is provisioned as part of a scale operation
-for a VM Scale Set, we suggest not using the Chef Infra Client
-extension. Instead pre-install Chef Infra Client onto the VM and/or use
-an image-based deployment process, such as
-[Packer](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/build-image-with-packer),
-to create the base VM disk image.
-
-The Microsoft VM agent on the machine executes all installations in
-parallel and these extensions can call 'locking' installation mechanisms
-such as Windows Installer (via installation of an MSI) or Apt updates. A
-timeout can occur if the total time for all of these activities exceeds
-5 minutes and in some regions that is not enough time to get the Chef
-Infra Client package and enable the Chef Infra Client extension.
-
+Installing the chef extension is no longer supported through the Azure Portal.
+It can be installed using other command line tools such as:
+  1. [Azure Powershell cmdlets](https://docs.chef.io/azure_powershell/)
+  2. [Azure Xplat CLI](https://docs.chef.io/azure_chef_cli/)
+  3. [Knife Azure Plugin](https://docs.chef.io/workstation/knife_azure/)
```

Comment:
```suggestion
The Chef Extension can no longer be installed through the Azure Portal.
It can be installed using other command line tools, such as:

1. [Azure Powershell cmdlets](/azure_powershell/)
1. [Azure CLI](/azure_chef_cli/)
1. [Knife Azure Plugin](/workstation/knife_azure/)
```